curriculum/js1/9.js

```diff
@@ -4,8 +4,10 @@
  * @returns {number}
  */
 
-const solution = (a) => {
-  return 0;
+const solution = (a=0) => {
+  if((a+1)%7===0)
```

Comment:
Please add a space after keywords like `if`, `for`, and `return`. In this case, you want `if ((a+1...`